cms/topics/tests/utils.py

```diff
@@ -0,0 +1,29 @@
+from typing import TYPE_CHECKING
+
+from django.urls import reverse
+from wagtail.test.utils.form_data import inline_formset, nested_form_data, rich_text, streamfield
+
+from cms.taxonomy.models import Topic
+
+if TYPE_CHECKING:
+    from django.test import Client
+
+
+def create_simple_topic_page(client: "Client", homepage_id: int) -> None:
```

Comment:
Nit: Could we add a docstring here explaining that this sends a post request etc..

Or alternatively we could make rename this to `send_request_to_create_simple_topic_page`, even if it is a mouthful, what do you think?